fontique/src/backend/coretext.rs

```diff
@@ -37,7 +41,17 @@ pub struct SystemFonts {
 
 impl SystemFonts {
     pub fn new() -> Self {
-        let scanned = scan::ScannedCollection::from_paths(Some("/System/Library/Fonts"), 8);
+        let paths = unsafe {
+            NSSearchPathForDirectoriesInDomains(
+                NSSearchPathDirectory::NSLibraryDirectory,
+                NSSearchPathDomainMask::NSAllDomainsMask,
+                Bool::YES,
+            )
+            .as_ref()
+            .iter()
+            .map(|p| format!("{p}/Fonts/"))
```

Comment:
An alternative would be:
```rust
p.stringByAppendingPathComponent(ns_string!("Fonts")).to_string()
```

But the solution you've chosen is probably fine too.

Reply:
If there's an advantage to that, I'm fine making this change ... I don't care either way though, just need a decision.